pkg/handlers/index/list.go

```diff
@@ -50,3 +49,37 @@ func List(c *gin.Context) {
 
 	c.JSON(http.StatusOK, items)
 }
+
+// @Id IndexNameList
+// @Summary List index Name
+// @Tags    IndexName
+// @Param   name  path  string  true  "IndexName"
```

Comment:
And the param `name` not in path, it's in `query`, please refer to:

https://github.com/swaggo/swag#param-type